evm/src/cpu/kernel/asm/core/create.asm

```diff
@@ -87,7 +87,7 @@ global create_common:
 
 run_constructor:
```

Comment:
Also with #961 you can delete `%set_new_ctx_parent_ctx` on line 96 (it's called in `%create_context`) above.